### ISSUE TITLE
java rpm download failed

### ISSUE THREAD

**Issue by kathirvel20** [srsp/ansible-oracle-java#20]
![image](https://user-images.githubusercontent.com/42608229/47290938-5ef14300-d61f-11e8-8922-421272cb811b.png)

**Comment by sydneybrokeit** [comment]
Every so often, Oracle seems to update their URLs.  Looks like it happened again.

**Comment by ionutzp** [comment]
👍 same here


> Sorry!
> --
>  
> In order to download products from Oracle Technology Network you must agree to the OTN license terms.
>  
>  
> Be sure that...
> Your browser has "cookies" and JavaScript enabled.
> You clicked on "Accept License" for the product you wish to download.
> You attempt the download within 30 minutes of accepting the license.
>

**Comment by kathirvel20** [comment]
where I need to change download link in the file.can you please mention details in readme file

**Comment by sydneybrokeit** [comment]
I was able to get around it by specifying version, subversion, build, and hash.

**Comment by srsp** [comment]
Oracle released jdk8_191. For some time now, they remove the old versions, so only the latest version is downloadable. I need to update the latest default version. Until then you can provide some vars in your playbook.

This should work (untested):

```
java_version: 8
java_subversion: 191
jdk_tarball_hash: 2787e4a523244c269598db4e85c51e0c
java_build: 12
```

**Comment by srsp** [comment]
I released version 2.19.2. Should work for jdk 8_u191 again. Others versions need to be downloaded beforehand and can then be be deployed by this role.

**Comment by kathirvel20** [comment]
thanks
